pkg/clusterapi/localcontrolplane.go

```diff
@@ -107,10 +107,15 @@ func (c *localControlPlane) Run(ctx context.Context) error {
 		return err
 	}
 
+	artifactsDirPath := filepath.Join(command.RootOpts.Dir, ArtifactsDir)
+	err = os.MkdirAll(artifactsDirPath, 0750)
+	if err != nil {
```

Comment:
Nit:
```suggestion
	if err := os.MkdirAll(artifactsDirPath, 0750;; err != nil {
```

Reply:
This was my first inclination but I did it this way to match Vince's style right above. :thinking: I will probably leave it as is.